workbench/src/renderer/components/ResourcesLinks/index.jsx

```diff
@@ -52,23 +56,24 @@ function handleClick(event) {
  * e.g. https://community.naturalcapitalproject.org/tag/carbon
  */
 export default function ResourcesTab(props) {
-  let userGuideURL = UG_ROOT;
-  let forumURL = FORUM_ROOT;
   const { docs, moduleName } = props;
+
+  let forumURL = FORUM_ROOT;
   const tagName = FORUM_TAGS[moduleName];
-  if (docs) {
-    const docsName = docs;
-    userGuideURL = `${UG_ROOT}/${docsName}#data-needs`;
-  }
   if (tagName) {
-    forumURL = `${FORUM_ROOT}/tags/${tagName}`;
+    forumURL = `${FORUM_ROOT}/tag/${tagName}`;
+  }
+
+  let userGuideURL = `${UG_ROOT}/index.html`;
+  if (docs) {
+    userGuideURL = `${UG_ROOT}/${docs}#data-needs`;

```

Comment:
When would `docs` be undefined?

Reply:
Probably never? But maybe that wasn't true when this was written. I updated this. It seems safe to not guard against this since we are literally validating `userGuideURL` for every model in this PR.